docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -32,8 +32,10 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && ln -s $(which python) /usr/local/bin/python
 
+# MXNet requires pip 19.3.1 due to being backwards compatible
+# with python2
 RUN pip --no-cache-dir install --upgrade \
-    pip \
+    pip==19.3.1 \
```

Comment:
Can you change this to "pip>=19.0,<20.0"?

Reply:
Is there a particular advantage in using "pip>=19.0,<20.0" vs. just 19.3.1? Since 19.3.1 is what seems to be in the current production containers as well?

Reply:
Also, the https://pypi.org/project/pip/ page seems to say that pip version 20 is compatible with python 2.7.

Reply:
I'm not sure if this is a Python 2 related issue or rather a binary wheel issue in general.

Reply:
@codewithsk If pip 20 really is targeted towards only py3, then we want to ensure that any future patches (however unlikely they may be) to pip 19 will be used in the images.

The issue in using pip==20.0.1 seems to be related to a package version python module that was used in the versioning of the MXNet wheel.

Reply:
@saimidu Does this apply to all other libraries as well or just pip? If not, why is this a special case, since == is the convention we've been following for other packages. If so, should all other pins be changed as well?

Reply:
@codewithsk Good point. We should definitely constrain the more quickly-changing packages like awscli, botocore, boto3, etc. rather than pinning them to a version. It looks like that is what is causing some CI build failures to occur.

Regardless, we will need to wait and see if pip or mxnet fix this issue (or if it is treated as a "feature" and not a "bug" by pip). Until then, `pip==19.3.1` works just as well.